web/viewer.html

```diff
@@ -81,8 +81,8 @@
             <button id="viewThumbnail" class="toolbarButton group toggled" title="Show Thumbnails" tabindex="2" data-l10n-id="thumbs">
                <span data-l10n-id="thumbs_label">Thumbnails</span>
             </button>
-            <button id="viewOutline" class="toolbarButton group" title="Show Document Outline" tabindex="3" data-l10n-id="outline">
-               <span data-l10n-id="outline_label">Document Outline</span>
+            <button id="viewOutline" class="toolbarButton group" title="Show Document Outline (double-click to expand/collapse all items)" tabindex="3" data-l10n-id="document_outline">
+                <span data-l10n-id="document_outline_label">Outline</span>
```

Comment:
The word `Document` has disappeared in this line, so that needs to be put back. Therefore, this line should be:

`<span data-l10n-id="document_outline_label">Document Outline</span>`